### PR TITLE
Headless chrome test sprint4

### LINKED ISSUE #228: Implement headless Chrome test for React UI for /users/{user_id}/tasks/{task_id}
## What:
This JavaScript test will go to the UI  https://nscstrdevusw2tuecommon.z5.web.core.windows.net/user/5/tasks/6  through headless chrome and take a screenshot

## How to use: 
- Required:
 Node and npm installed 

- From the repository root folder:
 $ npm i --save puppeteer
 
- Navigate to the JS file
 $ cd testing/ui-tests/ui-test-users-id-tasks-id.js   

- run the js code through node   
 $ node ui-test-users-id-tasks-id.js     
Check the /ui-tests/results for the Screenshot

| Date | Activity| Time |
| :---:        |     :---:      |     :---: |
| 2/23   | Created task on Github     |  5 min  |
| 2/23   |   Researched Puppeteer/headless chrome |  1.5 hrs  |
|  2/28  | Researched Puppeteer    | 1 hrs    |
|  2/28  |  Implemented Headless test | 0.5 hrs    |
| 3/1 | Debating about the results folder | 0.5 hrs |

### PR DESCRIPTION
**Pull request by asiagromasia**

### CODE REVIEW

File: testing/ui-tests/ui-test-users-id-tasks-id.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    (async () => {
+      const browser = await puppeteer.launch({headless: true});
+      const page = await browser.newPage();
+    
+      await page.goto('https://nscstrdevusw2tuecommon.z5.web.core.windows.net/user/5/tasks/6');
+      
+      let date = new Date();
+      let stringDate = date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate() + '-' + date.getHours() + ':' + date.getMinutes() + ':' + date.getSeconds();
+      await page.screenshot({path: './results/ad440-users-5-tasks-6-screenshot' + stringDate + '.png'});
+      console.log('Screenshot was just taken...');
```
Comment (sunshinejen) [inline testing/ui-tests/ui-test-users-id-tasks-id.js:11]: Nice! Looks great. Can we remove the other files that are not part of your task?

**Review by sunshinejen — CHANGES_REQUESTED** [review]

File: testing/ui-tests/ui-test-users-id-tasks-id.js (hunk 1 above)
Comment (asiagromasia) [reply testing/ui-tests/ui-test-users-id-tasks-id.js:11]: It was done automatically.

File: testing/ui-tests/ui-test-users-id-tasks-id.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer');
+
+    (async () => {
+      const browser = await puppeteer.launch({headless: true});
```
Comment (OyunominJuno) [inline testing/ui-tests/ui-test-users-id-tasks-id.js:4]: good job on making it headless

**Review by OyunominJuno — APPROVED** [review]
looks like this also pulled the name changes that Derek did, if those are most current in the development branch then your puppeteer code looks good and works

File: testing/ui-tests/ui-test-users-id-tasks-id.js (hunk 2 above)
Comment (sunshinejen) [reply testing/ui-tests/ui-test-users-id-tasks-id.js:4]: FYI puppeteer runs headless by default! While it is totally fine to write `headless: true`, if you omit it, this also works. 
https://developers.google.com/web/tools/puppeteer/get-started

**Review by sunshinejen — APPROVED** [review]

**Pull request merged**